bin/curriculum/import_hoc_lesson_plan.rb

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env ruby
```

Comment:
this file is mostly copied from https://github.com/code-dot-org/code-dot-org/blob/staging/bin/curriculum/import_unit_details.rb . because we do not plan to maintain this code once we are done with the imports, and because we aren't currently investing in automated tests for these scripts, I did not make an effort to extract shared helper methods.